src/pages/Details/Details.js

```diff
@@ -1,8 +1,65 @@
 import React, { Component } from 'react';
 import './Details.scss';
 
+import AddToCart from './AddToCart';
+import ProductInfo from './ProductInfo';
+import ProductDetails from './ProductDetails';
+import Tasting from './Tasting';
+
 export default class Details extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      productInfo: {},
+    };
+  }
+
+  componentDidMount() {
+    fetch('/data/DetailTest2.json')
+      .then(res => res.json())
+      .then(res => {
+        const productInfo = res.Result;
+
+        this.setState({ productInfo });
+      });
+  }
+
   render() {
-    return <div>Hello, Details!</div>;
+    const { productInfo } = this.state;
+
+    return (
+      <div className="details-wrapper">
+        {Object.keys(productInfo).length && (
+          <ProductInfo
+            key={productInfo.id}
+            hashtag={productInfo.hash}
+            awards={productInfo.awards}
+            name={productInfo.name}
+            grade={productInfo.grade}
+            image={productInfo.image}
+            price={productInfo.price}
+          />
+        )}
```

Comment:
```suggestion
        {Object.keys(productInfo).length && (
          <ProductInfo
            productInfo={productInfo}
          />
        )}
```
props로 넘길때는 이렇게 데이터 전체를 보내주시면 가독성이 좋아집니다!
그렇게 된다면 하위 컴포넌트 부분에서는 구조분해할당을 통해 사용하셔야겠죠?!